src/main/flight/pid.c

```diff
@@ -939,22 +940,26 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
     for (int axis = FD_ROLL; axis <= FD_YAW; ++axis) {
 
         float currentPidSetpoint = getSetpointRate(axis);
+        float rawSetpoint = getRawSetpoint(axis);
         if (pidRuntime.maxVelocity[axis]) {
             currentPidSetpoint = accelerationLimit(axis, currentPidSetpoint);
+            rawSetpoint = accelerationLimit(axis, rawSetpoint);
         }
         // Yaw control is GYRO based, direct sticks control is applied to rate PID
         // When Race Mode is active PITCH control is also GYRO based in level or horizon mode
 #if defined(USE_ACC)
         if ((levelMode == LEVEL_MODE_R && axis == FD_ROLL)
             || (levelMode == LEVEL_MODE_RP && (axis == FD_ROLL || axis ==  FD_PITCH)) ) {
             currentPidSetpoint = pidLevel(axis, pidProfile, angleTrim, currentPidSetpoint, horizonLevelStrength);
+            rawSetpoint = pidLevel(axis, pidProfile, angleTrim, rawSetpoint, horizonLevelStrength);
```

Comment:
This code will run no different than the line above it because you are using getRcCommandDelta(axis) to calculate your current angle. This is only different if you are in horizon mode, and as both have filtering applied to it after the setpoint is calculated, it isn't really raw at all. It won't work as the feedforward code is currently expecting.

TLDR this extra line of code is essentially doing nothing, but adding extra CPU.

Reply:
That line is only there to update rawSetpoint ready for FF when in level mode. Could just set rawSetpoint=currentPidSetpoint but wasn't sure about horizon mode so decided to duplicate the line.